src/components/AutoComplete/AutoComplete.js

```diff
@@ -0,0 +1,39 @@
+import React, { memo, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import checkProps from '@jam3/react-check-extra-props';
+
+import typingAhead from 'typing-ahead';
+
+const AutoComplete = ({
+  data,
+  className,
+  autoCompleteResult
+}) => {
+  const [model, setModel] = useState(null);
+
+  function onInputChange(event) {
+    const inputValue = event.target.value;
```

Comment:
Does this have to be stored locally or can you do this inline below?

Reply:
Not anymore since we got the value variable directly